docs/deploying-airbyte/integrations/custom-image-registries.md

```diff
@@ -0,0 +1,53 @@
+---
+products: oss-community, oss-enterprise
+---
+
+import ContainerProviders from '@site/static/_docker_image_registries.md';
+
+# Custom image registry
+
+If your organization uses custom image registries, you can use them with Airbyte, too. Airbyte supports public image registries without secrets and private or public ones that use secrets.
+
+Implementing Airbyte this way has several advantages.
+
+- **Security**: Private custom image registries keep images in your network, reducing the risk of external threats.
+- **Access control**: You have more control over who can access and modify images.
+- **Compliance**: By keeping images in a controlled environment, it's easier to prove compliance with regulatory requirements for data storage and handling.
+
+## Use a public custom image registry without secrets
+
+1. Add images to your custom image registry. If you don't already use a custom image registry, see the docs for these common options.
```

Comment:
A couple of questions @abuchanan-airbyte 

1. What's the expectation for image locations when we look to a custom location? Do we expect that all images are located in one place (IE use the default or use your custom location, but not both)?

2. Is there an easy way to get the full list of images we expect to see in this location?

Reply:
> What's the expectation for image locations when we look to a custom location? Do we expect that all images are located in one place (IE use the default or use your custom location, but not both)?

If you only configure `global.image.registry = ghcr.io/alex`, then all platform and connector images will be pulled from that registry.

It's possible to override that behavior for specific parts. For example, you can create a custom docker connector and set a fully-qualified image name (i.e. it includes the registry domain) like `docker.io/alex/my-connector`.

It's also possible to override specific platform images in the helm charts, such as `webapp.image.registry = docker.io/foo`.

> Is there an easy way to get the full list of images we expect to see in this location? 

Ya good question, I forgot to cover this in our meeting. For now, we recommend users use `abctl` to get a list of platform images for a particular version. For example, the following command will show all the images for the latest version:

```bash
> abctl images manifest
airbyte/bootloader:1.3.0
airbyte/connector-builder-server:1.3.0
airbyte/connector-sidecar:1.3.0
airbyte/container-orchestrator:1.3.0
airbyte/cron:1.3.0
airbyte/db:1.3.0
airbyte/mc:latest
airbyte/server:1.3.0
airbyte/webapp:1.3.0
airbyte/worker:1.3.0
airbyte/workload-api-server:1.3.0
airbyte/workload-init-container:1.3.0
airbyte/workload-launcher:1.3.0
bitnami/kubectl:1.28.9
busybox:1.35
busybox:latest
curlimages/curl:8.1.1
minio/minio:RELEASE.2023-11-20T22-40-07Z
temporalio/auto-setup:1.23.0
```

With that list, users can figure out how to re-tag and push those images to their registry. Here's what I would do:
```
abctl images manifest | xargs -L1 -I{} docker tag {} ghcr.io/foo/{} && docker push ghcr.io/foo/{}
```

`abctl images manifest` also takes some arguments that can affect the output, such as `--chart-version` and `--values`. `abctl` probably deserves its own section in the docs that describes this features.